servers/graphql-kotlin-server/src/main/kotlin/com/expediagroup/graphql/server/extensions/responseExtensions.kt

```diff
@@ -36,12 +35,17 @@ fun ExecutionResult.toGraphQLResponse(): GraphQLResponse<*> {
 /**
  * Convert the graphql-java type to the common serializable type [GraphQLServerError]
  */
-fun GraphQLJavaError.toGraphQLKotlinType() = GraphQLServerError(
-    message.orEmpty(),
-    locations?.map { it.toGraphQLKotlinType() },
-    path,
-    extensions
-)
+fun graphql.GraphQLError.toGraphQLKotlinType(): GraphQLServerError {
+    val newExtensions = this.extensions?.toMutableMap() ?: mutableMapOf()
+    newExtensions.putIfAbsent("classification", this.errorType.toSpecification(this))
```

Comment:
classification in extension? i don't think this is "standard"

Reply:
This is info that is in the graphql-java error that is lost if we do a raw conversion

https://github.com/graphql-java/graphql-java/blob/479b7a43087ec997959e5c9ce3048842370c80fa/src/main/java/graphql/GraphqlErrorHelper.java#L42-L44

Reply:
If we didn't need to convert this to a `GraphQLServerError` and instead could just return a string we could use the built-in `GraphqlErrorHelper.toSpecification()`

Reply:
we can keep the logic but you will have to check if error type is not null as otherwise you will get NPE (see https://github.com/graphql-java/graphql-java/blob/master/src/main/java/graphql/GraphqlErrorHelper.java#L35)

Reply:
Resolved